ext/crates/fp/src/limb.rs

```diff
@@ -46,6 +187,29 @@ pub(crate) const fn entries_per_limb_const<const P: u32>() -> usize {
     BITS_PER_LIMB / bit_length_const::<P>()
 }
 
+/// This is identical to [`limb::number`], except that it's not const. Hopefully almost every method
+/// in the limb crate can be const once the matrix rewrite is in place.
```

Comment:
We can probably implement the const generics version in terms of this. This can probably be a const fn as well.

Reply:
Good idea, I replaced the code for `number`